xbmc/powermanagement/PowerManager.cpp

```diff
@@ -327,3 +333,25 @@ void CPowerManager::SettingOptionsShutdownStatesFiller(const CSetting *setting,
 #endif
   }
 }
+
+void CPowerManager::WaitForNet()
+{
+  CLog::Log(LOGNOTICE, "%s: Waiting for first network interface to come up", __FUNCTION__);
+
+  const static int maxLoopCount = 50;
+  const static int sleepTimeMs = 200;
+
+  for(unsigned i=0; i < maxLoopCount; ++i)
+  {
+    CNetworkInterface* pIface = g_application.getNetwork().GetFirstConnectedInterface();
```

Comment:
Consider replacing `CNetworkInterface*` with `const auto * const` if `GetName()` is `const` or `const auto` otherwise.